src/components/v2/shared/FundingCycleConfigurationDrawers/NFTDrawer/MaxSupplyFormItem.tsx

```diff
@@ -0,0 +1,50 @@
+import { t } from '@lingui/macro'
+import { Form, FormInstance } from 'antd'
+import InputAccessoryButton from 'components/shared/InputAccessoryButton'
+import FormattedNumberInput from 'components/shared/inputs/FormattedNumberInput'
+import TooltipLabel from 'components/shared/TooltipLabel'
+
+import { NFTFormFields } from './NFTRewardTierModal'
+
+export default function MaxSupplyFormItem({
+  form,
+}: {
+  form: FormInstance<NFTFormFields>
+}) {
+  const validateMaxSupply = () => {
+    const value = form.getFieldValue('maxSupply')
+    if (value === undefined || value <= 0) {
+      return Promise.reject(t`Maximum supply required`)
+    }
+    return Promise.resolve()
+  }
+
+  return (
+    <div style={{ display: 'flex' }}>
+      <Form.Item
+        name={'maxSupply'}
+        label={
+          <TooltipLabel
```

Comment:
**suggestion**: I'd suggest we don't need the tooltip if we're already using the `extra` - lets do one or the other.

If the text is _essential_ to the user making a decision, then it should be in `extra`. Otherwise a tooltip is fine.